README.md

```diff
@@ -61,6 +61,25 @@ docker-compose start
 - Run `npm run dev` for development and `npm run start` for production.
 - Open your browser and navigate to http://localhost:5000
 
+### Example
+  This is the simple Example how BUB2(Book Uploader Bot) works.
```

Comment:
I can see incorrect capitalization throughout the section. Please fix them. You can use `` for highlighting purposes if incase you have capitalized for highlighting.

Reply:
@rkpattnaik780  please review it and tell me if it is right or wrong.